### DISCUSSION TITLE
Why is it not recognized when the generated bitcoin address is placed in https://coinfaucet.eu/en/btc-testnet to obtain test coins?

### DISCUSSION THREAD

**Discussion by 375372716** [trustwallet/wallet-core#2388] [Q&A]
Why is it not recognized when the generated bitcoin address is placed in https://coinfaucet.eu/en/btc-testnet to obtain test coins?

**Comment by 375372716** [comment]
How does Bitcoin switch to Segregated Witness and normal address types?

**Comment by hewigovens** [comment]
We don't support testnet address now

**Reply by 375372716** [reply]
In addition to the transaction sample codes of Bitcoin and Ethereum, are there any other transaction sample codes? Such as DOT, EOS

**Comment by 375372716** [comment]
Testnet addresses are not supported now? Do you want to use the official coin test program?
